### PR TITLE
Upgrade base64 to 0.21

### PR DESCRIPTION
**Pull request by viirya**
# Which issue does this PR close?

Closes #.

# Rationale for this change
 

# What changes are included in this PR?

# Are there any user-facing changes?

### CODE REVIEW

File: parquet/src/record/api.rs [hunk 1]
```diff
@@ -17,6 +17,10 @@
 
 //! Contains Row enum that is used to represent record in Rust.
 
+#[cfg(any(feature = "json", test))]
```
Comment (tustvold) [inline parquet/src/record/api.rs:20]: Perhaps we could move these into the function to avoid the macro shenanigans

**Review by tustvold — APPROVED** [review]
I wonder if we could use the prelude and the prefixed versions to reduce the code verbosity?

**Comment by viirya** [comment]
What "the prelude and the prefixed versions" is?

File: parquet/src/record/api.rs (hunk 1 above)
Comment (viirya) [reply parquet/src/record/api.rs:20]: Sounds good. Let me try to add a function for that.

**Comment by viirya** [comment]
Use `prelude` now.

**Review by tustvold — APPROVED** [review]

**Pull request merged**